var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -14,6 +14,8 @@ class Hypre(Package):
 
     # hypre does not know how to build shared libraries on Darwin
     variant('shared', default=sys.platform!='darwin', description="Build shared library version (disables static library)")
+    # SuperluDist have conflicting headers with those in Hypre
+    variant('internal-superlu', default=True, description="Use internal Superlu routines")
```

Comment:
good to know!

p.s. here is the description in config
```
--with-superlu          Use internal SuperLU routines.
```
Hypre also supports CMake installation and there don't even ported this option, i.e. CMakeList contains
```
#   --with-examples         Build example codes.
#   --with-fei              Use internal FEI routines.
#   --with-superlu          Use internal SuperLU routines.
#   --with-mli              Use MLI
```
and that's the only mentioning of SuperLU there.